apps/dashboard/src/components/dashboard-layout.tsx

```diff
@@ -1,29 +1,27 @@
 import { ReactNode } from 'react';
-import { UserProfile } from '@/components/user-profile';
-import { InboxButton } from '@/components/inbox-button';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { IntercomProvider } from 'react-use-intercom';
```

Comment:
some imported libraries error out with TS saying that the types path is not configured properly in the exported packages

Reply:
Is there no official sdk from the intercom team?

Reply:
yep, they suggest using their script; but the above library is what we use in the Web app and works well there with TS